### ISSUE TITLE
Использовать импорт xml для альтернативных проектов

### ISSUE THREAD

**Issue by ilyapishchulin** [VKCOM/icons#19]
Предлагаю сделать возможным импортировать xml для корректной работы, например, в React Native предпочтительны такие типы. С Гита устаешь копировать код

**Comment by fedorov-xyz** [comment]
Привет, сейчас можно импортировать напрямую svg файлы иконок. Это решает задачу?

**Comment by ilyapishchulin** [comment]
Нет, например, в React Native (iOS) плохо с svg и там лучше использовать xml через <SvgXml/>
